.github/workflows/deploy.yml

```diff
@@ -25,8 +25,11 @@ jobs:
       - name: Install dependencies
         run: |
           npm ci
-      - name: Setup mock git user
-        run: |
-          git config --global user.email "gmsa@fhir.com"
-          git config --global user.name "Unused account"
-      - run: npm run deploy
+      - run: npm run predeploy
```

Comment:
Yeah because the deploy creates a local commit so we'd need to setup a github user within the action which is a massive pain. Instead we build it and then let the specific github action for deployment handle it